tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py

```diff
@@ -19,19 +19,27 @@
 import grpc
 
 from framework import xds_k8s_testcase
+from framework.helpers import skips
 
 logger = logging.getLogger(__name__)
 flags.adopt_module_key_flags(xds_k8s_testcase)
 
 # Type aliases
 _XdsTestServer = xds_k8s_testcase.XdsTestServer
 _XdsTestClient = xds_k8s_testcase.XdsTestClient
+_Lang = skips.Lang
 
 _EXPECTED_STATUS = grpc.StatusCode.DATA_LOSS
 
 
 class CustomLbTest(xds_k8s_testcase.RegularXdsKubernetesTestCase):
 
+    @staticmethod
+    def is_supported(config: skips.TestConfig) -> bool:
+        if config.client_lang in _Lang.JAVA:
```

Comment:
Minor: let's add a quick comment explaining why it's only enabled in Java `v1.47.x`.
Usually I link https://github.com/grpc/grpc/blob/master/doc/grpc_xds_features.md, but I couldn't find your feature there. Should it be added?

Reply:
I added a comment, but I didn't mention this doc, will find out if it should get updated.